module2/projects/bulk_discounts.md

```diff
@@ -0,0 +1,143 @@
+
+**Example 3**
+
+* Merchant A has two Bulk Discounts
+    * Bulk Discount A is 20% off 10 items
+    * Bulk Discount B is 30% off 15 items
+* Invoice A includes two of Merchant A's items
+    * Item A is ordered in a quantity of 12
+    * Item B is ordered in a quantity of 15
+
+In this example, Item A should discounted at 20% off, and Item B should discounted at 30% off.
+
+**Example 4**
+
+* Merchant A has two Bulk Discounts
+    * Bulk Discount A is 20% off 10 items
+    * Bulk Discount B is 15% off 15 items
+* Invoice A includes two of Merchant A's items
+    * Item A is ordered in a quantity of 12
+    * Item B is ordered in a quantity of 15
+
+In this example, Both Item A and Item B should discounted at 20% off. Additionally, there is no scenario where Bulk Discount B can ever be applied.
+
+**Example 5**
+
+* Merchant A has two Bulk Discounts
+    * Bulk Discount A is 20% off 10 items
+    * Bulk Discount B is 30% off 15 items
+* Merchant B has no Bulk Discounts
+* Invoice A includes two of Merchant A's items
+    * Item A1 is ordered in a quantity of 12
+    * Item A2 is ordered in a quantity of 15
+* Invoice A also includes one of Merchant B's items
+    * Item B is ordered in a quantity of 15
+
+In this example, Item A1 should discounted at 20% off, and Item A2 should discounted at 30% off. Item B should not be discounted.
+
+## Required Features
+
+* Merchants should have full CRUD functionality over their bulk discounts including index and show pages for discounts
+* Merchant invoice show page should reflect discounted total
+* Merchant invoice show page should show which discounts were applied including links to the discount show page.
+* Admin invoice show page should reflect discounted total
+
+
+## User Stories
+
+```
+Merchant Bulk Discounts Index
+
+As a merchant
+When I visit my merchant dashboard
+Then I see a link to view all my discounts
+When I click this link
+Then I am taken to my bulk discounts index page
+Where I see all of my bulk discounts including their
+percentage discount and quantity thresholds
+And each bulk discount listed includes a link to its show page
+```
+
+```
+Merchant Bulk Discounts Show
+
+As a merchant
+When I visit my bulk discount show page
+Then I see the bulk discounts quantity and price
+```
+
+```
+Merchant Invoice Show Page: Total Revenue includes discounts
+
+As a merchant
+When I visit my merchant invoice show page
+Then I see that the total revenue for my merchant includes bulk discounts in the calculation
+```
+
+```
+Merchant Invoice Show Page: Link to applied discounts
+
+As a merchant
+When I visit my merchant invoice show page
+Next to each invoice item I see a link to the show page for the bulk discount that was applied (if any)
+```
+
+```
+Admin Invoice Show Page: Total Revenue includes discounts
+
+As an admin
+When I visit an admin invoice show page
+Then I see that the total revenue includes bulk discounts in the calculation
+```
+
+## Extensions
```

Comment:
I think an open ended feature as an extension is a good call for students that want to stretch.